composeApp/src/commonMain/kotlin/org/ooni/probe/App.kt

```diff
@@ -111,6 +114,14 @@ fun App(
                 navController.navigate("add-descriptor/${deepLink.id}")
                 onDeeplinkHandled()
             }
+            is DeepLink.RunUrls -> {
+                navController.navigate(Screen.ChooseWebsites(deepLink.url).route)
+                onDeeplinkHandled()
+            }
+            DeepLink.Error -> {
+                snackbarHostState.showSnackbar(getString(Res.string.LoadingScreen_Runv2_Failure))
```

Comment:
The "Error" message isn't clear enough for the user to understand what happened. We should explain OONI Probe only supports URLs to test.